docs/openshift/deploy.md

```diff
@@ -117,3 +117,53 @@ Run `acs-engine generate examples/openshift.json`
 After couple of minutes, your OpenShift web console should be accessible at `https://${dnsprefix}.${location}.cloudapp.azure.com:8443/`. You can log in with `clusterUsername` and `clusterPassword` values you set in the `openshift.json` file or with your Azure account if you've used AAD integration.
 
 For next steps, see [getting started documentation](https://docs.openshift.org/latest/getting_started/index.html) on OpenShift website.
+
+## Custom VNET
+
+ACS Engine supports deploying into an existing VNET. Operators must specify the ARM path/id of Subnets for the `masterProfile` and  any `agentPoolProfiles`, as well as the master IP address in `firstConsecutiveStaticIP`. Additionally, to prevent source address NAT'ing within the VNET, we assign to the `vnetCidr` property in `masterProfile` the CIDR block that represents the usable address space in the existing VNET.
```

Comment:
Currently OpenShift clusters cannot be set up in the 172.30.0.0/16 range (https://github.com/Azure/acs-engine/issues/2998).  Please document this.